src/collectives_c.c

```diff
@@ -392,9 +392,14 @@ shmem_longdouble_min_to_all(long double *target, long double *source, int nreduc
 {
     SHMEM_ERR_CHECK_INITIALIZED();
 
-    shmem_internal_op_to_all(target, source, nreduce, sizeof(long double),
+    if(!shmem_long_dub_supported) {
```

Comment:
Don't worry for now.  I will generate a patch for you to review.
On Dec 10, 2015 11:37 AM, "Kayla Seager" <notifications@github.com> wrote:

> In src/collectives_c.c
> <https://github.com/regrant/sandia-shmem/pull/53#discussion_r47273621>:
>
> > @@ -392,9 +392,14 @@ shmem_longdouble_min_to_all(long double *target, long double *source, int nreduc
> >  {
> >      SHMEM_ERR_CHECK_INITIALIZED();
> >
> > -    shmem_internal_op_to_all(target, source, nreduce, sizeof(long double),
> > +    if(!shmem_long_dub_supported) {
>
> in this case I'd have to undo some op.h work I've done. I've specialized
> everything for only the long double case. What are the combinations we are
> are anticipating? Should I just re-generalize for any atomic combination
> case?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/regrant/sandia-shmem/pull/53/files#r47273621>.
>